[compile error] Viewer->updateCamera() PCL1.11.1 vs PCL1.12.0

Hi,
I would like to ask you for a help. I started to use PCL 1.12.0 everything works fine for me. Except command `viewer->updateCamera()`. When I used  PCL version 1.11.1 and mentioned command worked fine without errors for me. Did anything changed with this command? Should I add/link same new Additional Dependency?

Here is code example which worked with PCL1.11.1 but with PCL1.12.0 doesn't :

```cpp
#include <pcl/io/pcd_io.h>
#include <boost/thread/thread.hpp>
#include <pcl/visualization/pcl_visualizer.h>

int main(int argc, char** argv)
 {	
	pcl::PointCloud<pcl::PointXYZ>::Ptr cloud(new pcl::PointCloud<pcl::PointXYZ>);
	
	if (pcl::io::loadPCDFile<pcl::PointXYZ>("file.pcd", *cloud) != 0)
	{
		return -1;
	}

	boost::shared_ptr<pcl::visualization::PCLVisualizer> viewer(new pcl::visualization::PCLVisualizer("Viewer"));
	viewer->addPointCloud<pcl::PointXYZ>(cloud, "cloud");	
	viewer->setPointCloudRenderingProperties(pcl::visualization::PCL_VISUALIZER_POINT_SIZE, 2, "cloud");
	viewer->updateCamera();

	while (!viewer->wasStopped())
	{
		viewer->spinOnce(100);
		boost::this_thread::sleep(boost::posix_time::microseconds(100000));
	}

	return 0;
 } 
```
 Here is Error:

```
1>PCL_test.obj : error LNK2001: unresolved external symbol "public: void __cdecl pcl::visualization::PCLVisualizer::updateCamera(void)" (?updateCamera@PCLVisualizer@visualization@pcl@@QEAAXXZ)
1>C:\Users\trebu\source\repos\PCL_test\x64\Release\PCL_test.exe : fatal error LNK1120: 1 unresolved externals
1>Done building project "PCL_test.vcxproj" -- FAILED.
```

OS: Windows 10
Environment: Visual Studio 2019
Installer: PCL-1.12.0-AllInOne-msvc2019-win64.exe

--- a/visualization/include/pcl/visualization/pcl_visualizer.h
+++ b/visualization/include/pcl/visualization/pcl_visualizer.h
@@ -1807,10 +1807,6 @@ namespace pcl
         std::string
         getCameraFile () const;
 
-        /** \brief Update camera parameters and render. */
-        void
-        updateCamera ();
-
         /** \brief Reset camera parameters and render. */
         void
         resetCamera ();

--- a/visualization/include/pcl/visualization/pcl_visualizer.h
+++ b/visualization/include/pcl/visualization/pcl_visualizer.h
@@ -1807,6 +1807,11 @@ namespace pcl
         std::string
         getCameraFile () const;
 
+        /** \brief Update camera parameters and render. */
+        PCL_DEPRECATED(1,15,"updateCamera will be removed, as it does nothing.")
+        inline void
+        updateCamera () {};
+
         /** \brief Reset camera parameters and render. */
         void
         resetCamera ();

